dynatrace/environment_v2/service_level_objectives.py

```diff
@@ -0,0 +1,282 @@
+        :param metric_denominator: The total count metric (the denominator in rate calculation). Required when the useRateMetric is set to false.
+        :param evaluation_type: The evaluation type of the SLO.
+        :param filter_: The entity filter for the SLO evaluation. Use the syntax of entity selector.
+        :param custom_description: The custom description of the SLO.
+        :param enabled: The SLO is enabled (true) or disabled (false).
+
+        :returns Response: HTTP response for the request
+        """
+        slo = self.get(slo_id=slo_id)
+        params = slo._to_json()
+
+        for key, val in locals().items():
+            if key in ["self", "slo_id", "slo", "params"]:
+                continue
+            if val is not None:
+                key = arg_to_camelcase(key)
+                params[key] = val if isinstance(val, (str, int, float, bool)) else str(val)
+
+        return self.__http_client.make_request(path=f"{self.ENDPOINT}/{slo_id}", method="PUT", params=params)
+
+    def delete(self, slo_id: str) -> "Response":
+        """Deletes an SLO
+
+        :param slo_id: The ID of the existing SLO.
+
+        :returns Response: HTTP response for the request
+        """
+        return self.__http_client.make_request(path=f"{self.ENDPOINT}/{slo_id}", method="DELETE")
+
+
+class Slo(DynatraceObject):
+    def _create_from_raw_data(self, raw_element: Dict[str, Any]):
+        # required
+        self.name: str = raw_element.get("name")
+        self.id: str = raw_element.get("id")
+        self.target: float = raw_element.get("target")
+        self.warning: float = raw_element.get("warning")
+        self.timeframe: str = raw_element.get("timeframe")
+        self.status: SloStatus = SloStatus(raw_element.get("status"))
+        self.evaluation_type: SloEvaluationType = SloEvaluationType(raw_element.get("evaluationType"))
+        self.use_rate_metric: bool = raw_element.get("useRateMetric")
+
+        # optional
+        self.metric_rate: str = raw_element.get("metricRate")
+        self.metric_numerator: str = raw_element.get("metricNumerator")
+        self.metric_denominator: str = raw_element.get("metricDenominator")
+        self.error_budget: Optional[float] = raw_element.get("errorBudget", 0)
+        self.numerator_value: Optional[float] = raw_element.get("numeratorValue", 0)
+        self.denominator_value: Optional[float] = raw_element.get("denominatorValue", 0)
+        self.related_open_problems: Optional[int] = raw_element.get("relatedOpenProblems", 0)
+        self.evaluated_percentage: Optional[float] = raw_element.get("evaluatedPercentage", 0)
+        self.filter: Optional[str] = raw_element.get("filter")
+        self.enabled: Optional[bool] = raw_element.get("enabled", False)
+        self.description: Optional[str] = raw_element.get("description")
+        self.error: Optional[SloError] = SloError(raw_element.get("error", SloError.NONE))
+
+    def _to_json(self) -> Dict[str, Any]:
```

Comment:
Can you call this `to_json()`? This is already done in other places in the code so it keeps consistency. Also return the element immediately instead of assigning multiple times, example

```
return {
  "name": self.name,
  "target": self.target
}
```